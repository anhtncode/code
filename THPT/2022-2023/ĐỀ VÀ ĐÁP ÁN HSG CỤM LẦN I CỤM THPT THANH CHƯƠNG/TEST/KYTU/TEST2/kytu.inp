lkjhabc ASDF Fvd d fdf ffsnd abbc  dd d adbf sff f  fs fss f sfsf sjff sf sfbb fbffs DAAGDG FBFBMF SFKFHSF FFBNGND GGKH
