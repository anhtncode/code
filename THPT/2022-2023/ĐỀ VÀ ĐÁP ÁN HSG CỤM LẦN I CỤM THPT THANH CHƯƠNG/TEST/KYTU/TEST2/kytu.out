a:6
b:10
c:2
d:12
f:30
g:5
h:3
j:2
k:3
l:1
m:1
n:3
s:14
v:1
